### ISSUE TITLE
Could you tell me how to fine tune the pre training model ?

### ISSUE THREAD

**Issue by hermannjin** [Linzaer/Ultra-Light-Fast-Generic-Face-Detector-1MB#132]

**Comment by chenti2x** [comment]
I want to know too,thanks

**Comment by thinh77** [comment]
Add directory of checkpoint file to argument --resume like this
"parser.add_argument('--resume', default='./models/RFB-Epoch-135-Loss-1.5431.pth', type=str, help='Checkpoint state_dict file to resume training from')"
Then, you have to change the order of calls like this
net = create_net(num_classes)
    if args.resume:
        logging.info(f"Resume from the model {args.resume}")
        net.load(args.resume)
        # net.load_state_dict(torch.load(args.resume, map_location=lambda storage, loc: storage))
    elif args.base_net:
        logging.info(f"Init from base net {args.base_net}")
        net.init_from_base_net(args.base_net)
    elif args.pretrained_ssd:
        # logging.info(f"Init from pretrained ssd {args.pretrained_ssd}")
        net.init_from_pretrained_ssd(args.pretrained_ssd)
